From b20fa68aa49897f728f8d1873c71d31b990ff2fc Mon Sep 17 00:00:00 2001
From: 0xD34D <clark@scheffsblend.com>
Date: Sat, 11 Nov 2023 06:57:43 +0000
Subject: [PATCH 05/43] Add sensor_x and sensor_y to prtouch

This will allow for defining where the nozzle's position should be
located to be over the sensor
---
 klippy/extras/prtouch.py | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/klippy/extras/prtouch.py b/klippy/extras/prtouch.py
index 5823a0ce5..b1bcf87c2 100644
--- a/klippy/extras/prtouch.py
+++ b/klippy/extras/prtouch.py
@@ -17,6 +17,8 @@ class PRTouchCFG:
         self.pi_count = config.getint('pi_count', default=32, minval=16, maxval=128)
         self.min_hold = config.getint('min_hold', default=3000, minval=100, maxval=50000)
         self.max_hold = config.getint('max_hold', default=50000, minval=100, maxval=100000)
+        self.sensor_x = config.getfloat('sensor_x', minval=0, maxval=300)
+        self.sensor_y = config.getfloat('sensor_y', minval=0, maxval=300)
         self.hot_min_temp = config.getfloat('s_hot_min_temp', default=140, minval=80, maxval=200)
         self.hot_max_temp = config.getfloat('s_hot_max_temp', default=200, minval=180, maxval=300)
         self.bed_max_temp = config.getfloat('s_bed_max_temp', default=60, minval=45, maxval=100)
-- 
2.39.5 (Apple Git-154)

